Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -782,6 +792,7 @@ static _WKWebExtensionError toAPI(WebExtension::Error error)
     populateContentSecurityPolicyStringsIfNeeded();
     populateWebAccessibleResourcesIfNeeded();
     populateCommandsIfNeeded();
+    populateDeclarativeNetRequestPropertiesIfNeeded();
```

Comment:
can update the `// FIXME: <https://webkit.org/b/246493>` above since you've added the dnr errors